pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -13,13 +14,12 @@ import (
 	"github.com/aws/aws-sdk-go/service/cloudwatchlogs/cloudwatchlogsiface"
 	"github.com/grafana/grafana-plugin-sdk-go/backend"
 	"github.com/grafana/grafana-plugin-sdk-go/data"
-	"github.com/grafana/grafana/pkg/components/simplejson"
 	"golang.org/x/sync/errgroup"
 )
 
 const (
 	LimitExceededException = "LimitExceededException"
-	defaultLimit           = 10
+	defaultLimit           = int64(10)
```

Comment:
Not sure if unexported constants should start with a capital letter or not (think it's lower case @iwysiu ?). Just as long as this constant and LimitExceededException use the same convention I'm happy. :)

Reply:
Thanks! :sunglasses: